config/routes.rb

```diff
@@ -4,9 +4,12 @@
   get '/auth/google_oauth2/callback', to: 'sessions#create'
   get '/auth/failure', to:redirect('/')
   get '/login', to: 'sessions#new'
-  get '/logout', to: 'sessions#destroy'
+  delete '/logout', to: 'sessions#destroy'
   get '/dashboard', to: 'users#show'
   get '/search', to: 'search#index'
   get '/search/:id', to: 'search#show', as: :search_show
   post '/image', to: 'images#create'
+  get '/500', to: 'application#server_errors'
+  get '/400', to: 'application#search_errors'
+  get '(*url)', to: redirect('/404')
```

Comment:
This protects from the app errors when a random path is attempted by the user.

Reply:
Does this cover all random hardcoded urls that we don't have routes for?

Reply:
That's what it should do, yes.